### PR TITLE
changes for the DOMOS project

### PR DESCRIPTION
**Pull request by jgriesfeller**
Just a small PR to add the data location of the DOMOS project

### CODE REVIEW

**Comment by lewisblake** [comment]
Similar type of issue as #891 with different variables

**Comment by jgriesfeller** [comment]
The domos project is finished, so I want this in main-dev.
This PR changes some color scales optimised so that the contours are usable.

File: pyaerocom/aeroval/glob_defaults.py [hunk 1]
```diff
@@ -1,17 +1,141 @@
 #: Default variable ranges for web display
 var_ranges_defaults = {
     "default": {"scale": [0, 1.25, 2.5, 3.75, 5, 6.25, 7.5, 8.75, 10], "colmap": "coolwarm"},
-    "ang4487aer": {"scale": [0, 0.25, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2], "colmap": "coolwarm"},
+    "ang4487aer": {
+        # "scale": [0, 0.25, 0.5, 0.75, 1, 1.25, 1.5, 1.75, 2],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:5]: please remove dead code

File: pyaerocom/aeroval/glob_defaults.py [hunk 2]
```diff
@@ -1,17 +1,141 @@
...
+            1.7,
+            1.8,
+            1.9,
+            2.0,
+        ],
+        "colmap": "coolwarm",
+    },
     "od550aer": {
-        "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
+        # "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:34]: dead code

File: pyaerocom/aeroval/glob_defaults.py [hunk 3]
```diff
@@ -1,17 +1,141 @@
...
+            0.8,
+            0.90,
+            1.0,
+            2.0,
+        ],
         "colmap": "coolwarm",
     },
     "od550gt1aer": {
-        "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
+        # "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:88]: rm

File: pyaerocom/aeroval/glob_defaults.py [hunk 4]
```diff
@@ -1,17 +1,141 @@
...
+            0.7,
+            0.8,
+            0.90,
+            1.0,
+            2.0,
+        ],
+        "colmap": "coolwarm",
+    },
+    "od550dust": {
+        # "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:115]: rm

File: pyaerocom/aeroval/glob_defaults.py [hunk 5]
```diff
@@ -91,6 +215,25 @@
     "dryoxs": {"scale": [0, 1.25, 2.5, 3.75, 5, 6.25, 7.5, 8.75, 10], "colmap": "coolwarm"},
     "dryoxn": {"scale": [0, 1.25, 2.5, 3.75, 5, 6.25, 7.5, 8.75, 10], "colmap": "coolwarm"},
     "dryrdn": {"scale": [0, 1.25, 2.5, 3.75, 5, 6.25, 7.5, 8.75, 10], "colmap": "coolwarm"},
+    "depdust": {
+        "scale": [0.0, 0.2, 0.5, 1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0],
+        # "scale": [0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:220]: rm

File: pyaerocom/aeroval/glob_defaults.py [hunk 6]
```diff
@@ -91,6 +215,25 @@
...
+        "colmap": "coolwarm",
+    },
+    "drydust": {
+        # "scale": [0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:224]: rm

File: pyaerocom/aeroval/glob_defaults.py [hunk 7]
```diff
@@ -91,6 +215,25 @@
...
+        "scale": [0.0, 0.2, 0.5, 1.0, 2.0, 5.0, 10.0, 20.0, 50.0, 100.0, 200.0, 500.0, 1000.0],
+        "colmap": "coolwarm",
+    },
+    "wetdust": {
+        # "scale": [0.0, 10.0, 20.0, 30.0, 40.0, 50.0, 60.0, 70.0, 80.0, 90.0, 100.0],
```
Comment (lewisblake) [inline pyaerocom/aeroval/glob_defaults.py:229]: rm

**Review by lewisblake — CHANGES_REQUESTED** [review]
Overall looks good, just please remove some commented out code

File: pyaerocom/aeroval/glob_defaults.py (hunk 1 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py [hunk 8]
```diff
@@ -1,17 +1,141 @@
...
+            0.8,
+            0.90,
+            1.0,
+            2.0,
+        ],
         "colmap": "coolwarm",
     },
     "od550lt1aer": {
-        "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
+        # "scale": [0, 0.05, 0.1, 0.15, 0.20, 0.25, 0.30, 0.35, 0.40],
```
[the remark this answers is not in the document]
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 3 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 5 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 6 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 7 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 2 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/aeroval/glob_defaults.py (hunk 4 above)
Comment (jgriesfeller) [reply pyaerocom/aeroval/glob_defaults.py]: done

File: pyaerocom/griddeddata.py [hunk 9]
```diff
@@ -2532,10 +2532,13 @@ def std(self):
...
-        if self.longitude.bounds is None:
+        if not self.longitude.has_bounds():
             self.longitude.guess_bounds()
-        if self.latitude.bounds is None:
-            self.latitude.guess_bounds()
+        if not self.latitude.has_bounds():
+            try:  # needs at least 2 values to work
+                self.latitude.guess_bounds()
+            except ValueError:
+                pass
```
Comment (lewisblake) [inline pyaerocom/griddeddata.py:2541]: Could a test be written for this?

**Review by lewisblake — CHANGES_REQUESTED** [review]

File: pyaerocom/griddeddata.py (hunk 9 above)
Comment (avaldebe) [reply pyaerocom/griddeddata.py:2541]: According to the comment, it needs 2 values to work.
Why not to write it this?
``` python
            if self.latitude.size > 2
                self.latitude.guess_bounds()
```

Comment (jgriesfeller) [reply pyaerocom/griddeddata.py]: @avaldebe `self.latitude` is of type `DimCoord` which does not have the size property, the try statement is the shortest way to test (but probably not the most pyhtonic). May I leave it as it is?

Comment (jgriesfeller) [reply pyaerocom/griddeddata.py:2546]: @lewisblake a test has been added

File: tests/io/test_readgridded.py [hunk 10]
```diff
@@ -102,6 +102,14 @@ def test_ReadGridded_read_var(reader_tm5: ReadGridded):
     assert gridded.var_name == "new_var"
     assert isinstance(gridded, GriddedData)
 
+    gridded_single_point = gridded.sel(
+        latitude=0.0,
+        longitude=0.0,
+    )
+    with pytest.raises(ValueError):
+        gridded_single_point.latitude.guess_bounds()
+        gridded_single_point.longitude.guess_bounds()
```
Comment (avaldebe) [inline tests/io/test_readgridded.py:111]: Here you are only testintg the 1st exception.
You can test each exception individually like this

``` python
    with pytest.raises(ValueError):
        gridded_single_point.latitude.guess_bounds()
    with pytest.raises(ValueError):
        gridded_single_point.longitude.guess_bounds()
```

File: pyaerocom/griddeddata.py [hunk 11]
```diff
@@ -2532,10 +2532,18 @@ def std(self):
 
     def _check_lonlat_bounds(self):
         """Check if longitude and latitude bounds are set and if not, guess"""
-        if self.longitude.bounds is None:
-            self.longitude.guess_bounds()
-        if self.latitude.bounds is None:
-            self.latitude.guess_bounds()
+        if not self.longitude.has_bounds():
```
Comment (avaldebe) [inline pyaerocom/griddeddata.py:2535]: I do not think the test added really covers this section.
Here the exception is silenced, but the test relies on the exception beer raised.

**Review by avaldebe — CHANGES_REQUESTED** [review]
I do not think the exception really test the change to `_check_lonlat_bounds`

File: tests/io/test_readgridded.py (hunk 10 above)
Comment (jgriesfeller) [reply tests/io/test_readgridded.py:111]: applied  in latest version

File: pyaerocom/griddeddata.py (hunk 11 above)
Comment (jgriesfeller) [reply pyaerocom/griddeddata.py:2535]: yeah, the ValueError exception is actually raised in iris. 
```
python3.11/site-packages/iris/coords.py:2384: in guess_bounds
```
But I needed to the pyaerocom code to pass through that exception to make the analysis work. So it's not really testable.

**Comment by jgriesfeller** [comment]
Can the rviewers please check if they are fine with the comments and changes?

**Review by lewisblake — APPROVED** [review]

**Pull request merged**